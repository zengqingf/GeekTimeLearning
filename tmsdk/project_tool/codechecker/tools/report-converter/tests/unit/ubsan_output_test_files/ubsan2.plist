<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>diagnostics</key>
	<array>
		<dict>
			<key>category</key>
			<string>unknown</string>
			<key>check_name</key>
			<string>UndefinedBehaviorSanitizer</string>
			<key>description</key>
			<string>load of value 7, which is not a valid value for type 'enum E'</string>
			<key>issue_hash_content_of_line_in_context</key>
			<string>6fc06d3a54ca9af82b0bff6421294a9d</string>
			<key>location</key>
			<dict>
				<key>col</key>
				<integer>10</integer>
				<key>file</key>
				<integer>0</integer>
				<key>line</key>
				<integer>13</integer>
			</dict>
			<key>path</key>
			<array>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>event</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>10</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>13</integer>
					</dict>
					<key>message</key>
					<string>load of value 7, which is not a valid value for type 'enum E'</string>
				</dict>
			</array>
			<key>type</key>
			<string>ubsan</string>
		</dict>
		<dict>
			<key>category</key>
			<string>unknown</string>
			<key>check_name</key>
			<string>UndefinedBehaviorSanitizer</string>
			<key>description</key>
			<string>load of value 2, which is not a valid value for type 'bool'</string>
			<key>issue_hash_content_of_line_in_context</key>
			<string>10aa12392ae8df6c482bea716d2de069</string>
			<key>location</key>
			<dict>
				<key>col</key>
				<integer>7</integer>
				<key>file</key>
				<integer>0</integer>
				<key>line</key>
				<integer>21</integer>
			</dict>
			<key>path</key>
			<array>
				<dict>
					<key>depth</key>
					<integer>0</integer>
					<key>kind</key>
					<string>event</string>
					<key>location</key>
					<dict>
						<key>col</key>
						<integer>7</integer>
						<key>file</key>
						<integer>0</integer>
						<key>line</key>
						<integer>21</integer>
					</dict>
					<key>message</key>
					<string>load of value 2, which is not a valid value for type 'bool'</string>
				</dict>
			</array>
			<key>type</key>
			<string>ubsan</string>
		</dict>
	</array>
	<key>files</key>
	<array>
		<string>files/ubsan2.cpp</string>
	</array>
	<key>metadata</key>
	<dict>
		<key>analyzer</key>
		<dict>
			<key>name</key>
			<string>ubsan</string>
		</dict>
		<key>generated_by</key>
		<dict>
			<key>name</key>
			<string>report-converter</string>
			<key>version</key>
			<string>x.y.z</string>
		</dict>
	</dict>
</dict>
</plist>
