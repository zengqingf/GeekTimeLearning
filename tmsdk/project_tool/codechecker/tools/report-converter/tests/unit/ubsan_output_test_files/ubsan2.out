files/ubsan2.cpp:13:10: runtime error: load of value 7, which is not a valid value for type 'enum E'
files/ubsan2.cpp:21:7: runtime error: load of value 2, which is not a valid value for type 'bool'